sap-s4hana-soap/2.3/modules/ROOT/pages/sap-s4hana-soap-connector-reference.adoc

```diff
@@ -292,30 +330,30 @@ Defines a configuration for TLS, which can be used from both the client and serv
 |===
 
 [[TrustStore]]
-=== Truststore
+=== Trust Store
```

Comment:
trust store -> truststore and key store -> keystore

Reply:
Fixed.